content/blog/the-importance-of-ubiquitous-language/index.md

```diff
@@ -0,0 +1,181 @@
+
+When producing their artifacts, generally in the form of user stories, they should use ubiquitous language continuously and consistently.  An examples story may be:
+
+> As a customer, I want to be able to add products to my shopping cart, so that I can continue shopping until I'm ready to check out
+
+Let's now explore how this user story may make its way into the designs for this application.
+
+### Design
+
+As a frequent communicator with the business, being good stewards of the ubiquitous and business languages applies to deign the same way it does to product.  They additionally have a large artifact that they generate and maintain for use by the team, and that is the visual designs and user interviews. 
+
+For both artifacts it is important to stay consistent with the ubiquitous language terminology.  By using this language these two artifacts are uniquely positioned to be one of the best methods to iterate and refine the ubiquitous language.  At the end of the day, the design files produced are consumed by both the business and the engineering team.  They are consumed by the business when validating user flows and UI designs, and they are used by the engineering team when reviewing, prepping for, and ultimately implementing the UI and the associated backend elements.  Because it is consumed by and influenced by both of these groups of people, it provides great opportunity to validate the ubiquitous language as well.  If engineers have questions about the terminology used in the design, they may try to make sense of things they see in the design using different terminology or concepts.  Its important that these terms and concepts are de-conflicted with designs so we don't create a translation layer.  Similarly, as the user interviews are happening, if the users are saying things like "this term doesnt make sense here", or "I would have expected this to be labeled x" it's a sign that the ubiquitous language being used isn't valid, and should be iterated upon.
+
+If this validation is done effectively, it has a lot more power than just improving communication between members of the team.  It has validated the UI design and flow but also, to some extent, the software design of the domain objects.
+
+So we have created a design, that through several interviews, has also validated the ubiquitous language that we are going to be using.  Lets see now how this gets into the software.
```

Comment:
More direct: So we have created a design and validated the ubiquitous language we are going to be using through several interviews.